rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.4.7-release-notes.adoc

```diff
@@ -0,0 +1,13 @@
+= RPA Builder 6.4.7 Release Notes
+
+RPA Builder is the on-premise application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+This version is internal only. All of the features and fixed issues for this version are in version xref:rpa-home::release-notes/rpa-builder-6.4.8-release-notes.adoc[6.4.8].
+
+// == See Also
+include::partial$release-notes/rn-headings.adoc[tag=seeAlso,leveloffset=+1]
+
+* https://help.mulesoft.com/s/[MuleSoft Help Center]
```

Comment:
Please, remember that the external link should take the caret symbol:
* https://help.mulesoft.com/s/[MuleSoft Help Center^]